src/VisualStudio/VisualBasic/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -130,11 +130,16 @@
                 </StackPanel>
             </GroupBox>
             
-            <GroupBox x:Uid="ClassificationsGroupBox"
-                      Header="{x:Static local:AdvancedOptionPageStrings.Option_Classifications}">
+            <GroupBox x:Uid="EditorColorSchemeGroupBox"
+                      Header="{x:Static local:AdvancedOptionPageStrings.Option_Editor_Color_Scheme}">
                 <StackPanel>
-                    <CheckBox x:Name="Use_enhanced_colors"
-                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Use_enhanced_colors_for_C_and_Basic}" />
+                    <ComboBox x:Name="Editor_color_scheme" IsEditable="false">
+                        <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Color_Scheme_Enhanced}" Tag="{x:Static local:AdvancedOptionPageStrings.Color_Scheme_Enhanced_Tag}" />
+                        <ComboBoxItem Content="{x:Static local:AdvancedOptionPageStrings.Option_Color_Scheme_VisualStudio2017}" Tag="{x:Static local:AdvancedOptionPageStrings.Color_Scheme_VisualStudio2017_Tag}" />
+                    </ComboBox>
+                    <TextBlock x:Name="Custom_VS_Theme_Warning"
```

Comment:
Can we add an additional message if any impacted colors have been customized so they know they have to go to F&C to reset things to default if they really want everything from this sub-theme? Or something.